### PR TITLE
Featured Branch Merge to Apple Clone

### PR DESCRIPTION
**Pull request by MrkarlosM**
Want to merge this branch to the main branch, this branch includes:

- Navigation Bar:
  - Gradient background for divs.
  - Apple logo.
  - Search Bar.
  - Hover property for the menus of the Navigation Bar.

- Background images for the main section.

- Footer part with images of products as backgrounds inside anchor elements.

- Footer part for contact data of the company.
  - Using anchor elements.

- Footer part for other links, like Copyright and Privacy Policy.
  - Using anchor elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,75 @@
...
+                <div class="last-titles">&nbsp;</div>
+
+            </div>
+        </section>
+        <section class="product-section">
+
+            <div class="product-one">&nbsp;</div>
+            <div class="product-two">&nbsp;</div>
+            <div class="product-three">&nbsp;</div>
+            <div class="product-four">&nbsp;</div>
```
Comment (ebeagusamuel) [inline index.html]: - [x] It contains four elements which are links (to other articles or #). The four elements at the bottom of the page(i.e the products) should be links(`<a href="#">`)

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
#### Hi team :wave: 
You have done well with this project, but there are some issues that you will need to fix and I will be pointed them out below. 
## Changes to implement :writing_hand: 
- [x] Your README is OK but can be even better. You can provide more description of the project and can even put the link to the original project so anyone looking at the project will be able to access the page you cloned.
- [x] The PR title is not meaningful. Use a more meaningful PR title that provides more context about what the PR is about.
- [x] This is currently how your website looks like when I visit the link to the live project:
Screenshot from 2020-10-29 23-43-01
This is happening because of the way you specify the path to your images in your CSS. Currently, you are specifying the path like this `/src/images/your_verse_title_small.png` which will work normally in your local development environment but won't work in a production environment which is what s happening now. You should specify the path like this `../images/your_verse_hero.jpg`.

Please fix all the changes highlighted and when you are done, request for another review.
slack(@ebeagusamuel)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @MrkarlosM and @AhmedAmin90 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Please provide more description of the project as previous TSE suggested.
- [x] I tested your project on my local machine it is working well. Please update your live link

### Footer
- [ ] Please fix the flag image issue in footer as prious TSE explained.

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/)|[Clone Page](https://raw.githack.com/AhmedAmin90/Apple-clone/featured/index.html)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: index.html [hunk 2]
```diff
@@ -1 +1,75 @@
...
+            <div class="right-top-footer">
+                <ul>
+                    <li> <a href="#">Site Map |</a> </li>
+                    <li> <a href="#">Hot News |</a>  </li>
+                    <li> <a href="#">RSS Feeds |</a>  </li>
+                    <li> <a href="#">Media Info |</a> </li>
+                    <li> <a href="#">Environment |</a>  </li>
+                    <li> <a href="#">Job Opportunities |</a></li>
+                    <li> <a href="#">Contact Us |</a> </li>
+                    <li><a href="#"> <img class="usa-flag" src="../images/image-(1).png" alt="flag" height="16" width="16"> </a></li>
```
Comment (umairarshadbutt) [inline index.html]: The correct path is `src="./src/images/image-(1).png"`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @MrkarlosM and @AhmedAmin90 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly compare the screenshots below 👇 and work on the suggested change.

### Footer
- [ ] Please fix the flag image issue in the footer as previous TSE explained.

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/)|[Clone Page](https://raw.githack.com/AhmedAmin90/Apple-clone/featured/index.html)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: index.html [hunk 3]
```diff
@@ -1 +1,75 @@
...
+                    <div class="nav-items hover"><a href="#">Store</a></div>
+                    <div class="nav-items hover"><a href="#">Mac</a></div>
+                    <div class="nav-items hover"><a href="#">iPod</a></div>
+                    <div class="nav-items hover"><a href="#">iPhone</a></div>
+                    <div class="nav-items hover"><a href="#">iPad</a></div>
+                    <div class="nav-items hover"><a href="#">iTunes</a></div>
+                    <div class="nav-items hover"><a href="#">Support</a></div>
+                    <div class="nav-items">
+                        <form >
+                        </form> <input type="text"> <i class="fas fa-search"></i></div>
```
Comment (layzrd) [inline index.html]: Kindly move the input field into the form.

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @MrkarlosM and @AhmedAmin90, Congratulations 🎊 👏, you did very well on this project. Kindly check and fix the following inline code review comment for enhancement.

Now, you can merge the PR to master and move ahead.
Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/)

**Pull request merged**